src/components/BreakdownSection/BreakdownSection.js

```diff
@@ -0,0 +1,27 @@
+import { Grid } from '@mui/material';
+import { Fragment } from 'react';
+import { Section } from '../UI/Section';
+import { CardMainNews } from './CardMainNews';
+import { breakdownNews } from './mockData';
+import { CardFlex } from '../UI/CardFlex';
+
+export const BreakdownSection = () => {
+  return (
+    <Section title='Breakdown'>
+      <Grid container spacing={4} alignItems='stretch'>
```

Comment:
alignItems='stretch' can be removed, it is a default value

Reply:
done :)